website/docs/integrations/github-actions/affected-stacks.md

```diff
@@ -23,9 +23,39 @@ Atmos checks component folders for changes first, marking all related components
 
 ## Usage Example
 
+### Config
+
+The action expects the atmos configuration file `atmos.yaml` to be present in the repository.
```

Comment:
please specify where in the repository it's expected to be or how to set that path